xls2xml/xls2xml/TSVReader.py

```diff
@@ -32,8 +33,6 @@ def __init__(self, tsv_filename, conf_filename, conf_key):
         :type tsv_filename: basestring
         :param conf_filename: configuration file path
         :type conf_filename: basestring
-        :param conf_key: first level key in the configuration to access that section
```

Comment:
Please explain the difference between conf_file and conf_key

Reply:
Fixed in 27f748bc